update/Pos/Update/Poll/Logic/Base.hs

```diff
@@ -65,33 +65,28 @@ import           Pos.Util.Util (leftToPanic)
 ----------------------------------------------------------------------------
 
 -- | Check whether BlockVersion is confirmed.
-isConfirmedBV :: MonadPollRead m => BlockVersion -> m Bool
-isConfirmedBV = fmap (maybe False bvsIsConfirmed) . getBVState
+isCompetingBV :: MonadPollRead m => BlockVersion -> m Bool
+isCompetingBV = fmap (maybe False bvsIsConfirmed) . getBVState
```

Comment:
`Active` may become adopted as well, but `active` is not competing.

Reply:
`Active` is the status of update proposal, but competing is block version state.

are you just talking about renaming this function?

Reply:
_Perhaps_ we should slightly modify this definition
> By the way, we call block version competing if it might become adopted.

For instance, version 100.500.228 can be adopted, but only if many different things happen. We probably shouldn't say that it implies that the corresponding proposal is confirmed, but instead make a requirement that there must be a confirmed proposal with this block version.

Reply:
done